contents/handbook/people/hiring-process/index.md

```diff
@@ -39,7 +39,7 @@ All of our recruiting is done in-house, and we do not work with external agencie
 
 ### Deciding to hire
 
-[‘Talent Compounds’](/handbook/company/values#talent-compounds) is one of our values here at PostHog. We think carefully about each new role and the complexity it introduces to the organization. We also have an extremely high bar for the people we do hire! 
+[‘You're the driver’](/handbook/company/values#youre-the-driver) is one of our values here at PostHog. We think carefully about each new role and the complexity it introduces to the organization. We also have an extremely high bar for the people we do hire! 
```

Comment:
the `Talent Compounds` value is a bit lost or part of the you're the driver, happy to change if not

Reply:
You're the driver:

> We hire people that are really great at their jobs, and get out of their way. There are no deadlines, very minimal coordination and you won't have us breathing down your neck.

so i think this change makes sense.